### DISCUSSION TITLE
What is the plan and direction for the CDMC Technical Spec Working Group?

### DISCUSSION THREAD

**Discussion by mcleo-d** [finos/compliant-financial-infrastructure#174] [FINOS and CDMC Roadmap]
The following questions have been raised regarding the planning and direction of the CDMC Technical Spec Working Group ...

1. What is the plan for the technical spec working group?
2. Who is involved in those conversations? 
3. Who is leading that effort? 
4. What is the timeline we're expecting for output from that group?

## Cloud Data Management Capabilities Framework
_Best Practices for Managing Data in the Cloud - [CDMC_Framework_V1.1.1.pdf](https://github.com/finos/compliant-financial-infrastructure/files/7650814/CDMC_Framework_V1.1.1.pdf)_

<a href="https://github.com/finos/compliant-financial-infrastructure/files/7650814/CDMC_Framework_V1.1.1.pdf"><img src="https://user-images.githubusercontent.com/6029572/144635425-fb5c92d5-209e-4b40-8bde-6362d19e6589.png" width="400"></a>

## CDMC Controls Procedures and Test Specifications
_[CDMC Controls Procedures and Test Specifications v1.1.1.pdf](https://github.com/finos/compliant-financial-infrastructure/files/7651679/CDMC.Controls.Procedures.and.Test.Specifications.v1.1.1.pdf)_

<a href="https://github.com/finos/compliant-financial-infrastructure/files/7651679/CDMC.Controls.Procedures.and.Test.Specifications.v1.1.1.pdf"><img src="https://user-images.githubusercontent.com/6029572/144655402-207a37b7-b7cb-475d-bc2d-c422ffbf5f09.png" width="400"></a>

## CDMC Key Controls and Automations
_[CDMC Key Controls and Automations - V1.1.1.pdf](https://github.com/finos/compliant-financial-infrastructure/files/7651698/CDMC.Key.Controls.and.Automations.-.V1.1.1.pdf)_

<a href="https://github.com/finos/compliant-financial-infrastructure/files/7651698/CDMC.Key.Controls.and.Automations.-.V1.1.1.pdf"><img src="https://user-images.githubusercontent.com/6029572/144655655-2c66ba9e-b977-4b8e-a32c-11d53e056cd1.png" width="400"></a>

**Comment by mcleo-d** [comment]
Count me in 👋🏻

**Comment by mtronci** [comment]
Happy to contribute to this item

**Comment by eddie-knight** [comment]
Please include me on any conversations related to this :+1:

**Comment by mcleo-d** [comment]
@peter-thomas-db @abdullahgarcia @thinkl33t - I'll count you in too 😁

**Comment by mcleo-d** [comment]
Hi all 👋🏻 

The latest version of the [CDMC_Framework_V1.1.1.pdf](https://github.com/finos/compliant-financial-infrastructure/files/7650814/CDMC_Framework_V1.1.1.pdf), [CDMC Controls Procedures and Test Specifications v1.1.1.pdf](https://github.com/finos/compliant-financial-infrastructure/files/7651679/CDMC.Controls.Procedures.and.Test.Specifications.v1.1.1.pdf) and [CDMC Key Controls and Automations - V1.1.1.pdf](https://github.com/finos/compliant-financial-infrastructure/files/7651698/CDMC.Key.Controls.and.Automations.-.V1.1.1.pdf) documents have been added to the main body of this discussion https://github.com/finos/compliant-financial-infrastructure/discussions/174#discussion-3715503

Many thanks,

James 🚀

**Comment by mcleo-d** [comment]
I would like to share the following question asked by Jorges in the Compliant Financial Infrastructure Slack below ...

https://finos-lf.slack.com/archives/C01RNUMC2DP/p1638892956151100

> Do we have any prelim/draft/rough document where we precisely define the meaning of "compliant infrastructure" ?
> What kind of "compliance" and what kind of "infrastructure" are we focusing on?
> I am browsing the CDMC Framework (thank you, James!)

It would be great to discuss if further definitions need to be created and shared.

**Comment by gibsocol** [comment]
I mentioned in the 17-Dec meeting that Snowflake had published configuration and test scripts that they used in KPMG's assessment of the Snowflake / Alation combo against the 14 key controls.

There are available here:  https://github.com/Snowflake-Labs/EDMC-CDMC-v1-14-Control_Mapping

**Comment by abdullahgarcia** [comment]
Commenting to "bring it back to life".

**Comment by thinkl33t** [comment]
We're going to modify https://github.com/finos/compliant-financial-infrastructure/pull/204 to better reflect this relationship now.

**Comment by eddie-knight** [comment]
A few conversations have been had during the community calls to determine that CDMC and CFI will both use CIS as baseline standards to streamline compatibility. Precise next steps are pending work that can currently only be executed by CDMC.

**Comment by eddie-knight** [comment]
Locking conversation as stale. If the CDMC collaboration issue is raised again in a community call we may come back to use this thread again at that time.
